### ISSUE TITLE
Character encoding issue

### ISSUE THREAD

**Issue by dannyhlee** [scalacenter/bloop#1405]
**Describe the bug**
Certain characters are not being written to text/csv file properly.  I noticed it with Asian (Korean, Japanese) and European language (Polish, German?) characters.  Some examples are:
```
"Gambit Kr�lowej","Poland","2020-11-21","01",26,0
"zdr�wko","Poland","2020-11-21","01",33,0
"Ot�?","Poland","2020-11-21","01",49,0
"Verschw�rungstheorien","Switzerland","2020-11-20","12",14,0
"gen�ve","Switzerland","2020-11-20","12",12,0
"L�ndern","Switzerland","2020-11-20","23",16,0
"bl�dsinn","Switzerland","2020-11-20","23",38,0
"????","Korea","2020-11-21","13",50,14184
"#????3D","Japan","2020-11-21","12",1,53992
```

I believe it's the encoding when the string is being written out.   It works properly in sbt from IntelliJ and CLI.  I came across the issue when I was testing out VS code and Metals.  I then installed the bloop server and tried compiling from the CLI and came across the same error so I thought I'd file a ticket.

**To Reproduce**
The code I'm running is available here [https://github.com/dannyhlee/pj2/tree/acf4b745f726d3d67d2faed5f56f4829f2f8c9b9](https://github.com/dannyhlee/pj2/tree/acf4b745f726d3d67d2faed5f56f4829f2f8c9b9)

The code errors out in the bloop CLI but finishes its run in VSC/Metals (I believe).  

A test JSON file is available here: [https://gist.github.com/dannyhlee/533af5762c3999cc7ee66f3e0474d7ee](https://gist.github.com/dannyhlee/533af5762c3999cc7ee66f3e0474d7ee)

I first ran the program in VS Code using Metals with two arguments (input and output.csv).

Using the bloop CLI I ran:
```
bloop clean
bloop compile pj2
bloop run pj2 --args "input" --args "output.csv"
```

**Expected behavior**
Foreign character to be saved correctly.

**Installation:**

- Operating system: Win 10 2004
- Editor: Visual Studio Code 1.51.1/Metals
- Metals version: v1.9.7
- bloop v1.4.5-6-4768184c
- Using Scala v2.12.8 and Zinc v1.3.0-M4+45-d4354be3
- Running on Java JDK v1.8.0_265 (C:\Program Files\AdoptOpenJDK\jdk-8.0.265.01-hotspot\jre)

**Additional context**

I'm not an experienced programmer and don't mean to be a bother, but I thought I'd let you know.  I'm not sure if perhaps this is on the 'todo' list and not high priority, but it was interesting enough for me to investigate and I share my findings with you.

**Search terms**

UTF8, question mark, marks, strange characters

**Comment by tgodzik** [comment]
Thanks for reporting! This seem to work correctly for me when running:
```scala
object Main {
  def main(args: Array[String]) {
    Files.write(Paths.get("./hello.txt"), "ąśź".getBytes())
  }
} 
```

what are you using to write to file? The needed `-Dfile.encoding=UTF-8` seems to be the default currently.

**Comment by dannyhlee** [comment]
Thanks for responding.  I loaded this code in VS code and tried running it and got the same response `???`.

```
package wordcount

import java.nio.file.{Files, Paths}

object WordCount {
  def main(args: Array[String]) {

    Files.write(Paths.get("./hello.txt"), "ąśź".getBytes())
  }
}
```
The only change Ive made is to scala 2.12.12.

If I run the code in scala repl it works fine.  I also tried disabling all plugins except for Scala(Metals) and Scala Syntax (official).  I looked in VScode settings but did not see `-Dfile.encoding=UTF-8`.  I was looking for a way to dump the json file for the settings but did not see one (possibly a feature for the extension, not sure if this is that useful).  

My plan is to try this snippet of code on another computer to see if its just this machine.
